qt/aqt/reviewer.py

```diff
@@ -883,6 +966,12 @@ def _contextMenu(self) -> list[Any]:
             [tr.studying_audio_and5s(), "7", self.on_seek_forward],
             [tr.studying_record_own_voice(), "Shift+V", self.onRecordVoice],
             [tr.studying_replay_own_voice(), "V", self.onReplayRecorded],
+            [
+                tr.actions_auto_advance(),
+                "",
```

Comment:
I tried `Shift+A` but it didn't work for me. Any suggestions?

Reply:
It's a bit confusing; it needs to be updated in multiple locations.

```diff
diff --git a/qt/aqt/reviewer.py b/qt/aqt/reviewer.py
index 856b5dd85..7df436318 100644
--- a/qt/aqt/reviewer.py
+++ b/qt/aqt/reviewer.py
@@ -590,6 +590,7 @@ class Reviewer:
             ("5", self.on_pause_audio),
             ("6", self.on_seek_backward),
             ("7", self.on_seek_forward),
+            ("Shift+A", self.toggle_auto_advance),
             *self.korean_shortcuts(),
         ]
 
@@ -968,7 +969,7 @@ timerStopped = false;
             [tr.studying_replay_own_voice(), "V", self.onReplayRecorded],
             [
                 tr.actions_auto_advance(),
-                "",
+                "Shift+A",
                 self.toggle_auto_advance,
                 dict(checked=self.auto_advance_enabled),
             ],
````